chrome/android/java/src/org/chromium/chrome/browser/init/AsyncInitializationActivity.java

```diff
@@ -269,8 +273,19 @@ public void onStartupFailure() {
     @Override
     @SuppressLint("MissingSuperCall")  // Called in onCreateInternal.
     protected final void onCreate(Bundle savedInstanceState) {
-        boolean runRenderer = true; // Toggle this when running browser process
-        if (runRenderer) {
+        boolean isCastanets = false; // Toggle this when running browser process
+
+        Context context = ContextUtils.getApplicationContext();
+        ApplicationInfo info;
```

Comment:
How about moving declaration of  |ApplicationInfo| to "try" statement?
|info| variable isn't used anywhere except for "try" statement.

Reply:
Done. thank you